tests/auto/make_VMTests.py

```diff
@@ -0,0 +1,215 @@
+# Usage:
+# git clone https://github.com/ethereum/tests
+# for i in tests/VMTests/*; do python3.6 make_VMTests.py $i > $MANTICORE/tests/EVM/VMTests/eth_`basename $i`.py; done
+#MANTICORE is manticore source folder
+from io import StringIO
+from binascii import unhexlify
+import pyevmasm as EVMAsm
+import hashlib
+
+def gen_test(testcase, filename, skip):
+
+    output = ''
+    if skip:
+        output += '''    @unittest.skip('Gas or performance related')\n'''
+
+    testname = (os.path.split(filename)[1].replace('-','_')).split('.')[0]
+    bytecode = unhexlify(testcase['exec']['code'][2:])
+    disassemble = ''
+    try:
+        #add silent=True when evmasm supports it
+        disassemble = '\n                  '.join(EVMAsm.disassemble(bytecode).split('\n'))
+    except Exception as e:
+        pass
+    sha256sum = hashlib.sha256(open(filename, 'rb').read()).hexdigest()
```

Comment:
Should use `with open(filename 'rb') as f: sha256sum = hashlib.sha256(f.read()).hexdigest()`

Side note: this line may use a lot of memory if the file is very large (see https://stackoverflow.com/questions/22058048/hashing-a-file-in-python).